Symfony/CS/Fixer/Contrib/LongArraySyntaxFixer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
```

Comment:
this could be at the Symfony level btw. We don't use the short array syntax (mainly because of our 5.3 support, but Sf 3.0 will keep using the long syntax at least until all 2.x versions are EOLed to simplify merging between branches).

What do you think @keradus